man/man8/zfs-mount-generator.8.in

```diff
@@ -45,6 +62,51 @@ history_event-zfs-list-cacher.sh .
 .RE
 .PP
 .sp
+.SS PROPERTIES
+The behavior of the generator script can be influenced by setting special properties on datasets:
+.sp
+.TP 4
+.BR org.open\-zfs.systemd:noauto
+Create and treat the mount unit as if it had
+.BR canmount=noauto
+set.
+.TP 4
+.BR org.open\-zfs.systemd:requires\-mounts\-for
+Space\-separated list of mountpoints to require to be mounted by this mount unit
+.TP 4
+.BR org.open\-zfs.systemd:before
+Space\-separated list of units to order after this mount unit
+.TP 4
+.BR org.open\-zfs.systemd:after
+Space\-separated list of units to order before this mount unit
+.TP 4
+.BR org.open\-zfs.systemd:wanted\-by
+Unit to add a dependency for this mount unit to.
```

Comment:
Maybe something like "The specified unit will gain a dependency on this mount unit."?